test/integration/QrochetQuacExt_test.jl

```diff
@@ -0,0 +1,20 @@
+@testset "Quac" begin
+    using Quac
+
+    @testset "QFT" begin
+        n = 3
+        qftcirc = Quac.Algorithms.QFT(n)
+        qftqtn = Quantum(qftcirc)
+
+        # correct number of inputs and outputs
+        @test ninputs(qftqtn) == n
+        @test noutputs(qftqtn) == n
+        @test socket(qftqtn) == Operator()
+        # all open indices are sites
+        siteinds = getindex.((qftqtn,), sites(qftqtn))
+        @test issetequal(inds(TensorNetwork(qftqtn), :open), siteinds)
+        # all inner indices are not sites
+        notsiteinds = filter(idx -> idx ∉ siteinds, keys(TensorNetwork(qftqtn).indexmap))
```

Comment:
Don't use `.indexmap`. Use `inds(TensorNetwork(qftqtn))`. In this case it's even easier with `setdiff`.
```suggestion
        notsiteinds = setdiff(inds(TensorNetwork(qftqtn)), siteinds)
```

Reply:
Actually, I think there is `inds(TensorNetwork(qftqtn), :inner)`. Can you try it?